usr/src/lib/brand/bhyve/zone/binstall

```diff
@@ -0,0 +1,32 @@
+#!/bin/ksh -p
+#
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2014 Joyent, Inc.  All rights reserved.
+# Copyright 2025 Edgecast Cloud LLC.
+#
+
+# Does this brand support reprovisioning?
+jst_reprovision="yes"
+
+# Is a template image optional?
+jst_tmplopt="yes"
+
+. /usr/lib/brand/jcommon/libhooks.ksh
+
+function jcommon_attach_hook
+{
+	jattach_kvm_final_setup
```

Comment:
The only problem I see here, is that jattach_kvm_final_setup in case of an error will output https://github.com/TritonDataCenter/illumos-joyent/blob/bad57c69ae70541c42a9026d00c312654add9758/usr/src/lib/brand/jcommon/libhooks.ksh#L29 
But if we see this error, it means we have bigger problems. Maybe now (if the impact is small) we could create a new separate function just for BHYVE, so that in the future, if this needs to be extended, it will be easier to isolate the changes.